packages/block-library/src/breadcrumbs/index.php

```diff
@@ -100,9 +100,25 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 			$show_terms = $attributes['prefersTaxonomy'];
 		}
 
-		// Build breadcrumb trail.
+		// Add post type archive link if applicable.
+		$post_type_object = get_post_type_object( $post_type );
+		$archive_link     = get_post_type_archive_link( $post_type );
```

Comment:
Should we even request the post type archive link if the post type doesn't have an archive?

Reply:
`get_post_type_archive_link` has all the checks we need and before checking the `has_archive`, has logic for `post` post type and the posts page settings. So it's fine doing this.